gm4_desire_lines/data/desire_lines/functions/path.mcfunction

```diff
@@ -0,0 +1,4 @@
+fill ~ ~-1 ~ ~ ~-1 ~ coarse_dirt replace dirt
+fill ~ ~-1 ~ ~ ~-1 ~ dirt replace grass_block
+fill ~ ~ ~ ~ ~ ~ air replace snow[layers=1]
+fill ~ ~ ~ ~ ~ ~ snow[layers=1] replace snow[layers=2]
```

Comment:
Unless flowers now break automatically, this is missing flower breaking. I suggest adding a `#desire_lines_breakable.json` block tag and replacing flower likes with air.